### PR TITLE
NO AUTO Strips atomic, hashing, temporal, date, index, neighbours, merge from full

### PR DESCRIPTION
**Pull request by ncordon**
Strips `atomic`, `hashing`, `temporal`, `date`, `index`, `neighbours`, `merge` procedures from full

### CODE REVIEW

File: common/src/main/java/apoc/date/DateUtils.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        if (unit == null) return TimeUnit.MILLISECONDS;
+
+        switch (unit.toLowerCase()) {
+            case "ms": case "milli":  case "millis": case "milliseconds": return TimeUnit.MILLISECONDS;
+            case "s":  case "second": case "seconds": return TimeUnit.SECONDS;
+            case "m":  case "minute": case "minutes": return TimeUnit.MINUTES;
+            case "h":  case "hour":   case "hours":   return TimeUnit.HOURS;
+            case "d":  case "day":    case "days":    return TimeUnit.DAYS;
+//			case "month":case "months": return TimeUnit.MONTHS;
+//			case "years":case "year": return TimeUnit.YEARS;
```
Comment (AzuObs) [inline common/src/main/java/apoc/date/DateUtils.java:18]: Could we remove these comments perhaps?

**Review by AzuObs — APPROVED** [review]
Doing the Lord's work! 💯

File: common/src/main/java/apoc/date/DateUtils.java (hunk 1 above)
Comment (ncordon) [reply common/src/main/java/apoc/date/DateUtils.java:18]: Yes, but I will clean it up in the next PR if you don't mind. Our CI is so flaky at the moment that I prefer to get the work in as soon as it's green

Comment (ncordon) [reply common/src/main/java/apoc/date/DateUtils.java:18]: Yes, I will do!

**Pull request merged**